yt_dlp/extractor/arte.py

```diff
@@ -45,6 +45,29 @@ class ArteTVIE(ArteTVBaseIE):
 
     _GEO_BYPASS = True
 
+    # Reference formerly available at: section 6.8 of
+    # <https://www.arte.tv/sites/en/corporate/files/complete-technical-guidelines-arte-geie-v1-07-1.pdf>
+
+    __LANG_MAP = {       # the RHS are not ISO codes, but French abbreviations
+        'fr': 'F',       # françois
+        'de': 'A',       # allemand
+        'en': 'E[ANG]',  # européen(?) [anglais]
+        'es': 'E[ESP]',  # européen(?) [espagnol]
+        'it': 'E[ITA]',  # européen(?) [italien]
+        'pl': 'E[POL]',  # européen(?) [polonais]
+    }
+
+    __VERSION_CODE_RE = re.compile(r'''(?x)
+        V
+        (?P<vo>O?)                        # original voice track
+        (?P<vlang>[FA]|E\[[A-Z]+\])?      # language of voice track
+        (?:
```

Comment:
Note to self:
```suggestion
        (?P<vaud>AUD|)      # audio description
        (?:
```

Should also add dispreference below.